src/modules/blocks/single-event/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+const { __ } = wp.i18n;
+const { useBlockProps } = wp.blockEditor;
+
+/**
+ * Small component to simplify some pseudo event blocks.
+ *
+ * @param width
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const FauxLine = ( {style ={}, ...props } ) => {
+    const divStyle = { height: 16, background: '#eee', margin: '18px 4px 18px 0' }
+
+    return ( <div style={{...divStyle, ...style}  } {...props} />    )
+}
+
+/**
+ * The Archive Events block used in Site Editor templates.
+ */
+export default {
+    id: 'tec/single-event',
+    title: __( 'Single Event', 'the-events-calendar' ),
+    icon: 'calendar-alt',
+    category: 'tribe-events',
+    keywords: [
+        __( 'Single Event', 'the-events-calendar' ),
+        __( 'The Events Calendar', 'the-events-calendar' ),
+    ],
+    edit: ( props ) => {
+        const { className, ...blockProps } = useBlockProps();
+
+        return (
+            <div className={ `${ className } ${ props.className }` } { ...blockProps }>
+                <h3>{ __( 'Event Title', 'the-events-calendar' ) }</h3>
+                <p>
+                    <strong>
+                    { __( 'EVENT DATE/TIME', 'the-events-calendar' ) }
+                </strong>
+                </p>
+
+                <FauxLine style={{marginLeft:34, marginRight:'25%'}} />
+                <FauxLine style={{marginRight:'25%'}} />
+                <FauxLine style={{marginRight:'25%'}} />
+                <button type={"button"} class="wp-block-button__link wp-element-button">Subscribe to Event</button>
```

Comment:
```suggestion
                <button type={"button"} class="wp-block-button__link wp-element-button">Subscribe to Event</button>
```
@EugeneKyale Ignore most of this PR, just wanted thoughts on how we ought to incorporate designs in these blocks. Not sure the future of some of this, but I know on the front-end the button design has the blue border around a white button. But I don't think we want to directly reuse the CSS from that, so for now I was just using the "bare theme" styles...?

Any opinion on which way to go on this for a v1?

Reply:
@stratease I think we should reuse the existing styles and have that as the default styling. This should give users a familiar base before applying any customisations.

Reply:
Going to do a style copy facade, not literally bringing in the CSS files.